src/ui/components/left_arrow.c

```diff
@@ -58,33 +60,21 @@ static void _render(component_t* component)
 static void _on_event(const event_t* event, component_t* component)
 {
     left_arrow_data_t* data = (left_arrow_data_t*)component->data;
-    const gestures_slider_data_t* slider_data = (const gestures_slider_data_t*)event->data;
+    if (event->data.source != data->location) {
+        return;
```

Comment:
Copied from https://github.com/BitBoxSwiss/bitbox02-firmware/pull/1607#discussion_r2390960245

Previously, `data->active = false` was set in the switch default case, now missing. Somehow, by touching the bottom slider while the arrow is active in the top slicer, active still seems to be reset to false and behave the same way as before, but I am not sure why.

In the show-mnemonic workflow, the arrows are on the bottom slider. Interestingly there, the arrow stays active even if touching the top slider, so the behavior is not symmetric.

Reply:
I think it still is there, see line 83.

Reply:
all the cases that `event->data.source != data->location` covers lead to `break` before and not to the `default` case in the switch.